layers/+lang/coq/README.org

```diff
@@ -70,6 +71,14 @@ font, or disable the feature by adding the following snippet to the your
 | ~SPC m p q~ | Quit prover                                                     |
 | ~SPC m p r~ | Retract buffer - rewinds and moves point to beginning of buffer |
 
+** Getting documentation
+
+| Key Binding | Description                                                     |
+|-------------+-----------------------------------------------------------------|
+| ~SPC m d .~ | Show documentation for whatever is below the cursor             |
```

Comment:
Documentation should be under the `SPC m h` prefix. Also this particular keybinding should be `SPC m h h`.